redirector/templates/primonator_form.html

```diff
@@ -0,0 +1,73 @@
+              </h1>
+            </div>
+            <div class="wrap-header-supp">
+              <a class="link-logo-mit" href="https://www.mit.edu"><span class="sr">MIT</span><svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" width="54" height="28" viewBox="0 0 54 28" enable-background="new 0 0 54 28" xml:space="preserve" class="logo-mit"><rect x="28.9" y="8.9" width="5.8" height="19.1" class="color"/><rect width="5.8" height="28"/><rect x="9.6" width="5.8" height="18.8"/><rect x="19.3" width="5.8" height="28"/><rect x="38.5" y="8.9" width="5.8" height="19.1"/><rect x="38.8" width="15.2" height="5.6"/><rect x="28.9" width="5.8" height="5.6"/></svg></a>
+            </div>
+          </header>
+        </div>
+      </div>
+      <div class="wrap-outer-content layout-band">
+        <div class="wrap-content">
+          <main id="content-main" class="content-main" role="main">
+            <p>
+              This very likely gives a stable Primo full record view link for a given
+              Aleph/Barton Permalink. However, if it does not we may not be able to fix
+              it as this is an experimental rather than official service.
+            </p>
+
+            <p>
+              <form class="form-horizontal" action="/primonator" method="GET">
+                <label for="aleph-url-1" class="field-label">Barton/Aleph Permalink</label>
+                <div class="form-group">
+                  <input id="aleph-url-1" class="field field-text form-input" name="aleph_link" placeholder="Enter Aleph Permalink" value="" type="text">
+                  <div class="aleph-url-submit form-action">
+                    <button type="submit" class="btn button-primary">Get Primo Link</button>
+                  </div>
+                </div>
+              </form>
+            </p>
+
+            <div class="well">
+              <p>
+                <ul class="list-unbulleted">
+                  <li><strong>Aleph Permalink</strong><br />{{ aleph_link }}</li>
+                  <li><strong>Aleph ID</strong><br />{{ aleph_id }}</li>
+                  <li><strong>Alma ID</strong><br />{{ alma_id }}</li>
+                  <li><strong>Alma Full Record Link</strong><br />{{ alma_link }}</li>
```

Comment:
Is this intentionally not a clickable link?

Reply:
Yup. I confirmed with the stakeholder that this was preferred for easier copy / paste.